specification/dimensions/chargesubcategory.md

```diff
@@ -0,0 +1,62 @@
+
+ChargeSubcategory MUST follow the requirements listed below:
+
+- The ChargeSubcategory MUST be present in the billing data.
+- ChargeSubcategory is of type String and MUST be one of the allowed values.
+- ChargeSubcategory MUST NOT be null or empty when ChargeType is "Usage" and the charge is covered by a commitment.
+  - When a usage charge is covered by a commitment, ChargeSubcategory MUST be "Commitment Used".
+  - When a commitment is not used within the committed period, ChargeSubcategory MUST be "Commitment Not Used" for the unused usage charge.
+- ChargeSubcategory MUST be null when ChargeType is "Usage" and is not covered by a commitment.
+- ChargeSubcategory MUST NOT be null or empty when ChargeType is "Adjustment".
+  - When an adjustment applies to a specific item, the corresponding FOCUS columns that identify that item MUST NOT be null and MUST match the applicable item details the adjustment pertains to.
+- ChargeSubcategory MUST be null when ChargeType is "Purchase" or "Tax".
+
+## Column ID
+
+ChargeSubcategory
+
+## Display Name
+
+Charge Subcategory
+
+## Description
+
+Indicates what kind of usage or adjustment the row represents.
+
+## Content Constraints
+
+| Constraint      | Value          |
+| :-------------- | :------------- |
+| Column required | True           |
+| Data type       | String         |
+| Allows nulls    | True           |
+| Value format    | list-of-values |
+
+Allowed values when ChargeType is `Usage`:
+
+| Value             | Description                                                                                                                                                                                                                                                                                                                                                |
+| :---------------- | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Commitment Used   | Usage charges that are associated with consumption of a commitment's underlying basis.                                                                                                                                              |
+| Commitment Not Used | Amortized usage charges for the portion of a commitment that has not been used. For example, if an organization has a commitment-based discount that is not fully utilized, the unused portion falls under this category. It highlights an area where the organization is not fully leveraging its commitments, which could be a lost cost-saving opportunity. |
```

Comment:
```suggestion
| Used Commitment   | Usage charges that are associated with consumption of a commitment's underlying basis.                                                                                                                                              |
| Unused Commitment | Amortized usage charges for the portion of a commitment that has not been used. For example, if an organization has a commitment-based discount that is not fully utilized, the unused portion falls under this category. It highlights an area where the organization is not fully leveraging its commitments, which could be a lost cost-saving opportunity. |
| On-Demand |  Usage charges that are associated with a commitment |
```

I think `Commitment Not Used` is very unclear if this means a commitment wasn't applied to this row or there was commitment that wasn't used in the row. I think while the `Unused` vs `Used` doesn't offer a large visual difference, the clarity is probably worth it. Adding an On-Demand here also means its easy to group usage costs into course groups.

Reply:
@mike-finopsorg I dont think adding On-Demand should be in the ChargeSubcategory as it is already used in chargetype (chargecategory) If we added on-demand then all chargetypes would be considered subcategories

Reply:
I do not see OnDemand in the Chargetype? https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/blob/working_draft/specification/dimensions/chargetype.md I think I am missing something that helps me understand your comment @thecloudman